notebooks/classification/mcmc_bnn_makemoons.ipynb

```diff
@@ -0,0 +1,462 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #2.    def bnn_log_joint(params, X, y, model):
</pre><p>It would be useful to provide a docstring for what params is</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/nipunbatra/bnn-zoo/pull/4/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='7'/>